src/GraphQL.Tests/Bugs/Bug1735.cs

```diff
@@ -0,0 +1,67 @@
+using System.Threading.Tasks;
+using GraphQL.SystemTextJson;
+using GraphQL.Types;
+using Shouldly;
+using Xunit;
+
+namespace GraphQL.Tests.Bugs
+{
+    // https://github.com/graphql-dotnet/graphql-dotnet/pulls/1735
+    public class Bug1735 : QueryTestBase<Bug1735Schema>
```

Comment:
There is no reported bug 1735.

Reply:
I mean the Bugs folder is for tests that verify reported bugs i.e. issues.

Reply:
Moved to `GraphQL.Tests.Execution.DocumentExecuterExceptions`